src/TrajectoryPlanner/src/TrajectoryGenerator.cpp

```diff
@@ -508,6 +521,31 @@ bool TrajectoryGenerator::getDCMPositionTrajectory(std::vector<iDynTree::Vector2
     return true;
 }
 
+bool TrajectoryGenerator::getDCMPositionTrajectoryAdjusted(std::vector<iDynTree::Vector2>& DCMPositionTrajectory)
+{
+    DCMPositionTrajectory = m_dcmGenerator->getDCMPosition();
+    return true;
+}
+
+bool TrajectoryGenerator::getDCMVelocityTrajectoryAdjusted(std::vector<iDynTree::Vector2>& DCMVelocityTrajectory)
```

Comment:
I think you can remove "Adjusted" from the name of these two methods. Now that you are using the same object for both cases, the term adjusted may create confusion.

Reply:
Done!